tests/scripts/all.sh

```diff
@@ -1197,6 +1197,73 @@ component_test_psa_external_rng_no_drbg_use_psa () {
     tests/ssl-opt.sh -f 'Default\|opaque'
 }
 
+component_test_full_no_md () {
+    msg "build: full minus MD"
+    scripts/config.py full
```

Comment:
Since all of X.509 and all of TLS get disabled, this ends up being pretty much the same as `test_crypto_full_no_md`. I haven't checked if the result is exactly identical, but it's close enough that testing a single configuration is enough.